tools/ci/maven-utils.sh

```diff
@@ -21,12 +21,10 @@ function run_mvn {
 		MVN_CMD="${M2_HOME}/bin/mvn"
 	fi
 
-	ARGS=$@
-	INVOCATION="$MVN_CMD $MVN_GLOBAL_OPTIONS $ARGS"
 	if [[ "$MVN_RUN_VERBOSE" != "false" ]]; then
-		echo "Invoking mvn with '$INVOCATION'"
+		echo "Invoking mvn with '$MVN_GLOBAL_OPTIONS ${@}'"
```

Comment:
From the commit message:
> This comes with a slight maintenance cost to maven-utils.sh#run_mvn, but given that we'll likely throw this out in 1.19 anyway (because there are better ways to implement this on later maven versions), we can bear that cost.

What are the costs? The quotes around `$@`? :thinking:  Is there a Jira issue for the "1.19 work"?

Reply:
> What are the costs?

The cost is `$MVN_GLOBAL_OPTIONS ${@}` being duplicated in the code.

> Is there a Jira issue for the "1.19 work"?

There is https://issues.apache.org/jira/browse/FLINK-32181, although it lacks a subtasks for this specific item.